packages/devtools_app/lib/src/screens/memory/framework/offline_data/offline_data.dart

```diff
@@ -38,20 +28,32 @@ class OfflineMemoryData {
     );
   }
 
+  static Map<String, Object?> toJson(OfflineMemoryData data) => {
+        _Json.selectedTab: data.selectedTab,
+        _Json.diffData: data.diff.toJson(),
+        _Json.profileData: data.profile.toJson(),
+        _Json.chartData: data.chart.toJson(),
+        _Json.classFilter: data.profile.classFilter.value.toJson(),
```

Comment:
instead of having these consts be in their own class _Json, can we add the constants to the class they are associated with `OfflineMemoryData`?

Reply:
It is more readable for me to have these consts grouped under on namespace.